app/Services/DailyService.php

```
@@ -0,0 +1,172 @@
+
+            return $response->successful();
+        } catch (Exception $e) {
+            Log::error('Daily room deletion failed: ' . $e->getMessage());
+            return false;
+        }
+    }
+
+    /**
+     * Create a consultation room for an appointment
+     */
+    public function createConsultationRoom(int $appointmentId, string $doctorName, string $patientName): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$doctorName'. (Unused Code Rules)

(UnusedFormalParameter)